fixtures/format.php

```diff
@@ -9,12 +9,16 @@ class TestClass
 {
     public $testProperty;
 
-                public function testMethod($testParameter)
+    /**
+     * @param $testParameter description
+     */
+    public function     testMethod(   $testParameter)
+
     {
         $testVariable = 123;
-        
-                if (  empty($testParameter)){
-                    echo 'Empty';
+
+        if (  empty($testParameter))    {
+            echo 'Empty';
         }
     }
 }
```

Comment:
CS-Fixer works in different way than Pretty Printer. PP is converting ast into source code according to some rules. CS-Fixer search for invalid part of code and fix just that part. Looks that not all parts of source code are covered by cs-fixer and for example "namespace       TestNamespace;" wont be converted to "namespace TestNamespace;". I still need to check if its by design or its a bug.